code/game/objects/items/devices/techno_tribalism.dm

```diff
@@ -0,0 +1,157 @@
+/obj/item/device/techno_tribalism
+	name = "Techno-Tribalism Enforcer"
+	desc = "Feeds on rare tools, tool mods and other tech items. After feeded enough, will produce a strange technological part, that will usefull to train your skills overtime."
+	icon = 'icons/obj/faction_item.dmi'
+	icon_state = "techno_tribalism"
+	item_state = "techno_tribalism"
+	origin_tech = list(TECH_MATERIAL = 8, TECH_ENGINEERING = 7, TECH_POWER = 2)
+	price_tag = 20000
+	var/list/oddity_stats = list(STAT_MEC = 0, STAT_COG = 0, STAT_BIO = 0, STAT_ROB = 0, STAT_TGH = 0, STAT_VIG = 0)
+	var/last_produce = -30 MINUTES
+	var/items_count = 0
+	var/max_count = 5
+	var/cooldown = 30 MINUTES
+
+/obj/item/device/techno_tribalism/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(items_count < max_count)
+		if(istype(W, /obj/item/weapon/tool))
+
+			if(W.tool_qualities)
+
+				for(var/qualitie in W.tool_qualities)
+
+					if(W.tool_qualities[qualitie] >= 15)
```

Comment:
You better check for for quality being more then 35 as well here, to ignore normal items and only feed advanced ones.

Also there is no such word is ``qualitie``.